src/styles/blocks/_container.scss

```diff
@@ -0,0 +1,13 @@
+.main {
```

Comment:
[BEM] - create a separate file per each BEM block styles `that have the same name as the block`

rename file